tools/odbc/api_info.cpp

```diff
@@ -40,7 +40,7 @@ const std::vector<SQLUSMALLINT> ApiInfo::ODBC3_SUPPORTED_FUNCTIONS = {
     SQL_API_SQLROWCOUNT,       SQL_API_SQLGETDIAGFIELD, SQL_API_SQLGETDIAGREC,     SQL_API_SQLGETFUNCTIONS,
     SQL_API_SQLBINDPARAMETER,  SQL_API_SQLGETDATA,      SQL_API_SQLFETCH,          SQL_API_SQLEXECUTE,
     SQL_API_SQLNUMRESULTCOLS,  SQL_API_SQLGETTYPEINFO,  SQL_API_SQLBINDCOL,        SQL_API_SQLCANCEL,
-    SQL_API_SQLNUMPARAMS};
+    SQL_API_SQLNUMPARAMS,      SQL_API_SQLMORERESULTS};
 
```

Comment:
You need to added the new supported function into the switch of `ApiInfo::GetFunctions`, e.g.:

```
switch (function_id) {
.
.
.
case SQL_API_SQLCANCEL:
case SQL_API_SQLNUMPARAMS:
case SQL_API_SQLMORERESULTS:
```
This is used when the DriverManager checks the support for individual functions.

Reply:
Could `ApiInfo::GetFunctions` be modified to look into `ApiInfo::ODBC3_SUPPORTED_FUNCTIONS` to check if the function is supported or not? Then the code wouldn't need to be duplicated in two places.

Perhaps `ApiInfo::ODBC3_SUPPORTED_FUNCTIONS` could also be turned into an `unordered_set`, instead of a `vector`, to make this faster?